substrate/test-utils/runtime/Cargo.toml

```diff
@@ -16,37 +16,43 @@ workspace = true
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
+sp-application-crypto = { features = [
+  "serde",
+  "bls-experimental",
```

Comment:
@coax1d you enforcing "bls-experimental" here no matter if it is requested in cargo build or not is problamatic and results in inconsistency. You should only have application_crypto/bls_experimental activated if it is activated for the crate (runtime).

Reply:
fixed in https://github.com/w3f/bls/commit/004e1aee0b7c492797e98e8ed036b2adb3b0f600